pundit.gemspec

```diff
@@ -19,12 +20,4 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
 
   gem.add_dependency "activesupport", ">= 3.0.0"
-  gem.add_development_dependency "activemodel", ">= 3.0.0"
```

Comment:
@thomasklemm Oh, since all development takes place within the Bundler context, and we have better control of versions needed etc using Gemfiles, we can use a Group in Bundler (`:development`) and keep development-related things there.